Source/gmenu.cpp

```diff
@@ -160,7 +161,19 @@ void gmenu_draw()
 	if (sgpCurrentMenu) {
 		if (dword_63447C)
 			dword_63447C(sgpCurrentMenu);
+
+#ifdef HELLFIRE
+		ticks = GetTickCount();
+		if ((int)(ticks - LogoAnim_tick) > 25) {
+		    LogoAnim_frame++;
+			if (LogoAnim_frame > 16)
+				LogoAnim_frame = 1;
+			LogoAnim_tick = ticks;
+		}
+		CelDecodeOnly(169, 102 + SCREEN_Y, sgpLogo, LogoAnim_frame, 430); //TODO: figure out what 169 means and apply defs?

```

Comment:
Maybe (SCREEN_WIDTH - 430) / 2 + SCREEN_X?

Reply:
considering 430 is the width, this sounds good to me lol

Reply:
updated